DESCRIPTION

```diff
@@ -1,14 +1,14 @@
-Package: SKELETON
+Package: hySpc.read.spe
 Type: Package
 Version: 0.0.0.9000
-Title: Package's Title
-Description: Short Description of the package.
+Title: File Import Functions for Spectra produced by Princeton Instruments spectrometers
 Authors@R: c(
     person("Claudia",   "Beleites",  role = c("aut"), email = "Claudia.Beleites@chemometrix.gmbh"),
```

Comment:
I think, several more people (incl. Roman Kiselev) must be included in the list of authors.

Reply:
I add Roman, but I'm not sure who else should I add.

Reply:
I think we should ask @cbeleites to be totally sure. I opened #4 for the discussion so we will be able to fix this in the future (after the issue is discussed). So after my notes are addressed in #3, I think we can merge #3.